From 81e16fc38127b9e117d65ca6f138a0e11780bb80 Mon Sep 17 00:00:00 2001
From: xyyx <xyyx@mail.ru>
Date: Sat, 24 Nov 2018 14:22:25 +0800
Subject: [PATCH 251/254] NetworkTraffic: Fix gravity

TextView.setText() has left gravity by default. It's ugly in statusbar

Change-Id: I7fac570597e925c47df45b788823bd959c972234
Signed-off-by: Josh Fox (XlxFoXxlX) <joshfox87@gmail.com>
Signed-off-by: Arghya Chanda <arghyac35@gmail.com>
Signed-off-by: Jayant-Deshmukh <jayantdeshmuk008@gmail.com>
Signed-off-by: DennySPb <dennyspb@gmail.com>
---
 .../android/systemui/statusbar/policy/NetworkTrafficSB.java    | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/packages/SystemUI/src/com/android/systemui/statusbar/policy/NetworkTrafficSB.java b/packages/SystemUI/src/com/android/systemui/statusbar/policy/NetworkTrafficSB.java
index a304acf80e3..88863c831da 100644
--- a/packages/SystemUI/src/com/android/systemui/statusbar/policy/NetworkTrafficSB.java
+++ b/packages/SystemUI/src/com/android/systemui/statusbar/policy/NetworkTrafficSB.java
@@ -26,6 +26,7 @@ import android.os.SystemClock;
 import android.provider.Settings;
 import android.util.AttributeSet;
 import android.util.TypedValue;
+import android.view.Gravity;
 import android.view.View;
 import android.widget.TextView;
 
@@ -108,6 +109,7 @@ public class NetworkTrafficSB extends TextView implements StatusIconDisplayable
 
                 // Update view if there's anything new to show
                 if (!output.contentEquals(getText())) {
+                    setGravity(Gravity.RIGHT | Gravity.CENTER_VERTICAL);
                     setTextSize(TypedValue.COMPLEX_UNIT_PX, (float)txtSize);
                     setText(output);
                 }
@@ -305,6 +307,7 @@ public class NetworkTrafficSB extends TextView implements StatusIconDisplayable
         txtImgPadding = resources.getDimensionPixelSize(R.dimen.net_traffic_multi_text_size);
         setTextSize(TypedValue.COMPLEX_UNIT_PX, (float)txtSize);
         setCompoundDrawablePadding(txtImgPadding);
+        setGravity(Gravity.RIGHT);
     }
 
     @Override
-- 
2.30.2

